src/encoded/commands/create_mapping_on_deploy.py

```diff
@@ -144,8 +145,13 @@ def main():
         env = app.registry.settings.get('env.name')
         if 'webprod' in env:
             if data_env != env:
-                log.info("looks like we are on staging, run create mapping without check first")
-                run_create_mapping(app, check_first=False, item_order=ITEM_INDEX_ORDER)
+                log.info("Looks like we are deploying staging, checking --wipe-es")
+                if args.wipe_es:
+                    log.info('--wipe-es specified - obliging')
+                    run_create_mapping(app, check_first=False, item_order=ITEM_INDEX_ORDER)
+                else:
+                    log.info('--wipe-es not specified - will not wipe ES')
+                    run_create_mapping(app, check_first=True, item_order=ITEM_INDEX_ORDER)
                 return
         # handle mastertest ... by blowing away all data first
         if 'mastertest' in env:
```

Comment:
Should this be an `elif`?  Assuming I'm not mis-reading here, the comment is worrisome in context of the above, and it seems possible a weird env name like `webprod-mastertest` could cause havoc, running everything twice, but in any case if the arms are disjoint normally anyway it seems worth making that clear.

Reply:
There's a call to log.info that says  "looks like we are NOT on staging or mastertest " that also looks like it belongs in some elif, too, and I'm less certain it's properly enumerating the cases there. This is messiness I do NOT expect to see cleaned up on this PR, as it feels like it would drift too far from cause. Even the `elif` question might be too much expansiveness. But I don't like this kind of control-flow vagueness of text message and haphazardness of expression on something so critical as wiping data.